packages/common-ui-web/src/components/TakePictureButton/index.ts

```diff
@@ -0,0 +1,2 @@
+export { TakePictureButton } from './TakePictureButton';
+export type { TakePictureButtonProps } from './hooks';
```

Comment:
I prefer the type exported *per member* like this :

```typescript
export { type TakePictureButtonProps } from './hooks';
```

It doesn't change a lot but it is just a preference